packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -263,16 +259,12 @@ Iterable<Method> buildTags(
                 ..name = toDartName(parameter.name)
                 ..required = parameterRequired
                 ..type = refer(result.nullableName);
-
-              if (parameter.schema!.$default != null) {
-                b.defaultTo = Code(valueToEscapedValue(result, parameter.schema!.$default!.toString()));
-              }
```

Comment:
This needs to stay to indicate the default value as it will be used.

Reply:
This is fundamentally broken.
If we want to support arbitrary objects to have default values this does not work.

The UriTemplate pr also needs the parameters to be built_collections as the serializer can't work with plain Lists or Maps.

It'll make semse once I push it tomorrow.